src/crypto/aes.rs

```diff
@@ -124,7 +125,7 @@ impl From<ring::error::Unspecified> for IronOxideErr {
 fn derive_key_from_password(password: &str, salt: [u8; PBKDF2_SALT_LEN]) -> [u8; AES_KEY_LEN] {
     let mut derived_key = [0u8; digest::SHA256_OUTPUT_LEN];
     pbkdf2::derive(
-        &digest::SHA256,
+        pbkdf2::PBKDF2_HMAC_SHA256,
```

Comment:
This is equivalent to the old value, correct?

Reply:
I expect so, but the old value gave an error of
`expected struct ring::pbkdf2::Algorithm, found struct ring::digest::Algorithm`
so I thought that this was what it wanted now

Reply:
Looks right to me.